transform/snowflake-dbt/models/mattermost/hourly/server_fact.sql

```diff
@@ -169,9 +193,14 @@ WITH sdd AS (
         , max(COALESCE(r.outgoing_webhooks, s.outgoing_webhooks)) as outgoing_webhooks
         , max(COALESCE(r.max_time, s.max_time)) AS max_timestamp
         , MAX(COALESCE(r.context_ip, r.context_request_ip)) AS last_ip_address
+        , MAX(board) AS boards
+        , MAX(_view) AS boards_views
```

Comment:
Why is this view with an underscore `_view`?

Reply:
The table originally has an underscore. No idea why its like that, I can update it, but if its being used downstream or anyone's code across MM, it will break.

Reply:
I see, yea we should look at making a change but not a priority for this PR